### ISSUE TITLE
File 'Accessibility-version.dll' not found or not a PE file

### ISSUE THREAD

**Issue by RussKie** [carlossanlop/DocsPortingTool#88]
An attempt to backport to `///` fails with the following error:

```
Diagnostic messages found in workspace.OpenProjectAsync - D:\Development\dotnet-winforms\src\System.Windows.Forms\src\System.Windows.Forms.csproj:
    Failure - Msbuild failed when processing the file 'D:\Development\dotnet-winforms\src\Accessibility\src\Accessibility.ilproj' with message:
        EXEC: (0, 0): File 'D:\Development\dotnet-winforms\artifacts\bin\Accessibility-version\Debug\net6.0\Accessibility-version.dll' not found or not a PE file
    Failure - Msbuild failed when processing the file 'D:\Development\dotnet-winforms\src\Accessibility\src\Accessibility.ilproj' with message:
        C:\Users\igveliko\.nuget\packages\microsoft.net.sdk.il\6.0.0-rc.2.21454.1\targets\Microsoft.NET.Sdk.IL.targets: (106, 5): The command ""C:\Users\igveliko\.nuget\packages\\runtime.win-x64.microsoft.netcore.ildasm\6.0.0-rc.2.21454.1\runtimes\win-x64\native/ildasm" "D:\Development\dotnet-winforms\artifacts\bin\Accessibility-version\Debug\net6.0\Accessibility-version.dll" /OUT="D:\Development\dotnet-winforms\artifacts\obj\Accessibility\Debug\net6.0\/Accessibility.ref.il"" exited with code 1.
```

Running the following command:
```
-CsProj D:\Development\dotnet-winforms\src\System.Windows.Forms\src\System.Windows.Forms.csproj -Docs D:\Development\dotnet-dotnet-api-docs\xml -Direction ToTripleSlash -IncludedAssemblies System.Windows.Forms.dll -IncludedNamespaces System.Windows.Forms
```

For the time being I've disabled the diagnostics as follows, but there should be a better way to ignore this error:
```diff
diff --git a/Libraries/ToTripleSlashPorter.cs b/Libraries/ToTripleSlashPorter.cs
index b4a3110..e02039f 100644
--- a/Libraries/ToTripleSlashPorter.cs
+++ b/Libraries/ToTripleSlashPorter.cs
@@ -474,7 +474,8 @@ namespace Libraries
                         Log.Error(msg);
                     }
 
+                    //Log.Finished();
+                    //throw new Exception("Diagnostic errors found.");
                 }
             }
         }
```
